frontend/src/index.js

```diff
@@ -4,7 +4,7 @@ import App from "./App";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+	<React.StrictMode>
```

Comment:
Just be aware that Strict mode has differences in local env and on f.e. prod, as far as I know.

Reply:
We will remove it, thanks.